src/caremote.c

```diff
@@ -197,6 +199,15 @@ CaRemote* ca_remote_unref(CaRemote *rr) {
         return mfree(rr);
 }
 
+int ca_remote_set_rate_limit(CaRemote *rr, size_t rate_limit) {
+        if (!rr)
+                return -EINVAL;
+
+        rr->rate_limit = rate_limit;
```

Comment:
so, right now "0" means "off" here, right?

To me it appears more natural to use UINT64_MAX as the right value for disabling the limiter, after all that's the closest value to infinity we can store, and a limit of infinity means no limit...